### ISSUE TITLE
[Feature Request] Allow changing default cursor

### ISSUE THREAD

**Issue by eduardz1** [pop-os/cosmic-settings#648]
In gnome I can change the default cursor under "Appearance -> Cursor" in the "GNOME Tweaks" app, it would be nice if it was possible directly in the COSMIC Settings app, perhaps under the "Desktop -> Appearance" settings.

It seems to be something the could be grouped in the, currently experimental, "Icon theme" section

**Comment by dmac-au** [comment]
I was able to set some consistency between gnome apps and cosmic apps by running:
```shell
gsettings set org.gnome.desktop.interface cursor-size 24
gsettings set org.gnome.desktop.interface cursor-theme Pop
```
So the Gnome app cursor is smaller and uses Pop's cursor, but I actually like the bigger 30px cursor size on my 1440p monitor so would also like to see this as an option, or at least a text based config so I can manually set it.

**Comment by gxanshu** [comment]
Anything struggling to get it in PopOS 24 or cosmic desktop try the following docs
https://www.reddit.com/r/pop_os/comments/1g31qof/cosmic_setting_a_cursor_theme/

**Comment by Milli189** [comment]
yeah the settings need to have an option to change the cursor theme

<img width="1116" height="834" alt="Image" src="https://github.com/user-attachments/assets/c1183581-3189-43bc-9f7b-2a261174579b" />

it has the option to change the icons theme so it should also have the option to change the mouse cursor

**Comment by omnireq** [comment]
More cursor customization will be coming as part of [Epoch 3](https://blog.system76.com/post/cosmic-epoch-2-and-3-roadmap) when you get svg cursors. In the meantime, for folks that need larger cursors you can put the following at the bottom of your ~/.profile. Make it 96 for an even larger cursor.

xrdb -merge Xcursor.theme: Cosmic
xrdb -merge Xcursor.size: 48
export XCURSOR_THEME=Cosmic
export XCURSOR_SIZE=48
